packages/hub/src/hub.ts

```diff
@@ -225,13 +226,56 @@ export class Hub implements HubInterface {
     top.scope.addBreadcrumb(finalBreadcrumb, Math.min(maxBreadcrumbs, MAX_BREADCRUMBS));
   }
 
+  /**
+   * @inheritDoc
+   */
+  public setUser(user: User | null): void {
+    const top = this.getStackTop();
+    if (!top.scope) {
+      return;
+    }
+    top.scope.setUser(user);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setTags(tags: { [key: string]: string }): void {
+    const top = this.getStackTop();
+    if (!top.scope) {
+      return;
+    }
+    top.scope.setTags(tags);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setExtras(extras: { [key: string]: any }): void {
```

Comment:
isnt this setExtra? and extra?

Reply:
Good point, before we only had `setExtra` on the scope which allowed only `key:value` `setExtras` allowed to set an object.

I will change that it accepts both.

Reply:
:+1: i think its fine though i would probably want to override the entire extra per old functionality (for other things folks should probably just use setContext)

Reply:
Maybe, we should even consider not even exposing `setExtra` at all anymore.
Could be too early tho, I mean it's still there in `configureScope` but not on `Sentry.set...`

Reply:
You cannot have multiple input types in Go, so I'd still need to have 2 methods for it